mithril-common/src/messages/cardano_transaction_snapshot_list.rs

```diff
@@ -62,19 +65,15 @@ mod tests {
 
     // Test the retro compatibility with possible future upgrades.
     #[test]
-    fn test_v1() {
+    fn test_v2() {
```

Comment:
This part always was clunky as those "versions" only existing in those tests names (there's something to do here).

Those tests needs to verify case such as:
- current version deserialization
- we updated the current version and want another test that check that:
  - we can parse json that contains now removed fields
  - we can parse json that contains renamed fields
  - we can parse json that miss newly added fields (using default values if available, else this would be a breaking change).